src/_data/team.yml

```diff
@@ -1000,3 +1000,15 @@
   linkedin: josewenzel
   email: jose.campos@codurance.com
   office: london
+
+- id: eloy_acosta
+  name: Omar Eloy Acosta Padron
+  image: /assets/custom/img/team/eloy_acosta.jpg
+  role: Software Craftsman
+  fullDescription:
+    <p>Eloy is constantly improving personal and professional skills for providing quality software solutions and systems.</p>
+    <p>His first contact with computers was in his childhood with the Epson QX 10.</p>
+    <p>Since obtained a degree of Licentiate in Computer Engineering by the ULPGC, he has been involved in managing and developing projects for several companies using a wide variety of technologies and practices.</p>
```

Comment:
This part doesn't read quite right. It should be 'Since **he** obtained a degree' or Since **obtaining** a degree'.

Reply:
Thanks Keith for looking at this.
I have update to correct the text.